README.md

```diff
@@ -1,45 +1,101 @@
+1. [Client Installation](#client-installation)
+1. [Contribute](#contribute)
+
+## Market API
+
+DEX services are offered via the **Market API**. The
+[DEX specification](spec/README.mediawiki) details the messaging and trading
+protocols required to use the Market API.
+
+In order to be accessible to the widest range of applications and languages, the
+Market API is accessed using JSON messages over WebSockets.
+A basic client application can be implemented in most popular programming
+languages with little more than the standard library.
+
+## Client Applications and the Core Package
+
+The initial DEX release also includes the client **Core** module, written in Go.
+**Core** offers an intuitive programmer interface, with methods for creating
+wallets, registering DEX accounts, viewing markets, and performing trades.
+
+**dcrdex** has two applications built with **Core**.
+
+The **browser-based GUI** (a.k.a. "the app") offers a familiar exchange
+experience in your browser. The app is really just a one-client web server that
+you run and connect to on the same machine. The market view allows you to see
+the market's order book in sorted lists or as a depth chart. You can place your
+order and monitor it's status in the same market view.
+
+The **dexcctl** utility enables trading via CLI. Commands are parsed and
+issued to **Core** for execution.
+
+## Using Decred DEX
+
+**The Decred DEX is in early stages of development, and should not be used to
+conduct trades on mainnet.** For those who would like to contribute, or just
+poke around and offer feedback, there are a number of ways to do that.
 
-## Using the DEX Server
+- [Run **dcrdex** and **dexc** on simnet](../../wiki/Simnet-Testing). Recommended for development.
+- [Run **dexc** on testnet](../../wiki/Testnet-Testing). Recommended for poking around.
+- [Run the test app server](../../wiki/Test-App-Server). Useful for GUI development, or just to try everything out without needing to create wallets or connect to a **dcrdex** server.
 
-The Decred DEX is in early stages of development, and should not be used to
-conduct trades on mainnet. Instructions below are tailored for running on
-testnet.
+## Server Installation
 
-### Dependencies
+#### Dependencies
 
 1. Linux
 2. [Go >= 1.13](https://golang.org/doc/install)
 3. [PostgreSQL 11+](https://www.postgresql.org/download/), [tuned](https://pgtune.leopard.in.ua/) and running.
 
-**Create the database** in a PostgreSQL `psql` terminal.
+#### Set up the database
+
+In a PostgreSQL `psql` terminal, run
 
 ```
 CREATE USER dcrdex WITH PASSWORD 'dexpass';
 CREATE DATABASE dcrdex_testnet OWNER dcrdex;
 ```
 
-**Generate a master public key** for collecting registration fees. This can be
+#### Generate a master public key
+
+The master public key is used for collecting registration fees. This can be
 done with [dcrwallet](https://github.com/decred/dcrwallet) using the
 `getmasterpubkey` RPC command. Place the pubkey string into a new DEX
```

Comment:
Make it obvious that it mustn't be the default account's master pub?
```suggestion
`getmasterpubkey ("account")` RPC command. Place the pubkey string into a new DEX
```

Reply:
Also, L105 below reads `# Mainnet extended pubkey`, should probably be `testnet extended pubkey` or just `extended pubkey` as we've earlier discouraged running on mainnet.